functests/utils/mcps/mcps.go

```diff
@@ -89,6 +89,18 @@ func GetConditionStatus(mcpName string, conditionType machineconfigv1.MachineCon
 	return corev1.ConditionUnknown
 }
 
+// GetConditionReason return the message of the given MCP
+func GetConditionReason(mcpName string, conditionType machineconfigv1.MachineConfigPoolConditionType) string {
```

Comment:
Just a question, how about combining two methods: GetConditionStatus() and GetConditionReason()
They are doing the same, just returning different parameters...

Reply:
This func doesn't exist anymore.